operator/internal/manifests/config.go

```diff
@@ -42,6 +42,11 @@ func LokiConfigMap(opt Options) (*corev1.ConfigMap, string, error) {
 
 // ConfigOptions converts Options to config.Options
 func ConfigOptions(opt Options) config.Options {
+	var rulerEnabled bool
+	if opt.Stack.Rules != nil && opt.Stack.Rules.Enabled {
+		rulerEnabled = true
+	}
```

Comment:
Convert to direct bool assignment?
```suggestion
	rulerEnabled := opt.Stack.Rules != nil && opt.Stack.Rules.Enabled
```